frontend/src/scenes/max/maxThreadLogic.tsx

```diff
@@ -78,6 +82,31 @@ export interface MaxThreadLogicProps {
     conversation?: ConversationDetail | null
 }
 
+/** Maps agent modes to the scenes that should trigger them */
+const SCENE_TO_AGENT_MODE: Record<AgentMode, Set<Scene>> = {
+    [AgentMode.SQL]: new Set([Scene.SQLEditor]),
```

Comment:
Also maybe these can be unified with `getModeOptions` so we have one source of truth for everything modes, frontend wise